src/util/precCompare.ml

```diff
@@ -129,6 +129,7 @@ struct
     |> List.filter (fun ((i1, _), (i2, _)) -> i1 <> i2)
     |> List.map (Tuple2.map snd snd)
     |> List.map (uncurry compare_dumps)
-    |> List.iter (fun (_, msg) -> ignore (Pretty.printf "%t\n" (fun () -> msg)));
-    ignore (Pretty.printf "\nTotal locations: %d\nTotal %s: %d\n" locations_count (Key.name ()) location_vars_count)
+    |> List.iter (fun (_, msg) -> Logs.info "%t" (fun () -> msg));
+    Logs.newline ();
+    Logs.info "Total locations: %d\nTotal %s: %d" locations_count (Key.name ()) location_vars_count
```

Comment:
This is the main output of the corresponding executable, why not just write to stout as usual?